src/app/components/BookList/components/Book/index.tsx

```diff
@@ -1,23 +1,27 @@
 import React from 'react';
-import { View, Text, Image } from 'react-native';
+import { View, Text, Image, TouchableOpacity } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
 import { BookProps } from '@interfaces/books';
 import imagePlaceholder from '@assets/img_book6.png';
+import ROUTES from '@constants/routes';
 
 import styles from './styles';
 
 const Book = ({ title, author, image }: BookProps) => {
   const sourceImage = image ? { uri: image } : imagePlaceholder;
+  const navigation = useNavigation();
+  const navigate = () => navigation.navigate(ROUTES.DETAIL);
```

Comment:
Use a more descriptive name, for example, `handleGoToDetail` or `handleNavigateToDetail` or something like that